src/components/MarkdownFlow/markdownFlow.css

```
@@ -5,11 +5,38 @@
   position: absolute;
   bottom: 20px;
   right: 20px;
-  border-radius: 100%;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: 32px;
+  height: 32px;
+  padding: 0;
+  border: 1px solid color-mix(in srgb, currentColor 20%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)